docs/OBOPURLDomain.md

```diff
@@ -1,11 +1,39 @@
 ---
 layout: doc
 id: OBOPURLDomain
-title: OBOPURLDomain
+title: OBO_PURL_Creation
```

Comment:
We can make this human-friendly and use spaces

(we should do this for all docs, they are camel case as the title was auto-generated from the filename originally)